docs/examples/data_management_framework/resources.rst

```diff
@@ -0,0 +1,65 @@
+Data Management Framework Resource Examples
+============================================
```

Comment:
Should we have some manner of cross-linking to https://idaes-pse.readthedocs.io/en/stable/user_guide/components/dmf/dmf-api.html?

Or perhaps more succinctly, do w need to reorganise things so that DMF docs are more easily found?